### ISSUE TITLE
Remote Desktop with Shadow mode

### ISSUE THREAD

**Issue by fanynek86** [Ylianst/MeshCentralRouter#106]
Hello gentlemen!

I would like to ask you...
Wouldn't it be possible to add the "Shadow mode" rdp startup parameter to MeshCentral? To use RDP like this:

https://www.informaticar.net/remote-desktop-with-shadow-mode-in-windows/

**Comment by fanynek86** [comment]
Thank you!

I'm glad I finally expressed myself in a meaningful way. If we connect to ordinary users, the session ID is always 1. In principle, you are right if there are more connected users.

**Comment by si458** [comment]
@fanynek86 what im thinking is,
when you click the RDP button inside the meshcentralrouter (this happens when u click the RDP option in the web ui),
the meshcentralrouter app will check for user sessions (like what the normal web ui does when u click connect),
then asks you 'do u want a normal console access, an additional rdp session, or shadow a user session,
if u select shadow, then it will say 'these are users i have which one do you want to shadow?',
then when u pick a user it will add the extra values `/shadow:<sessionID> /control /noConsentPrompt` etc where needed

**Comment by fanynek86** [comment]
@si458 Yes, you thought that beautifully! I didn't receive the email notification and I'm sick, so I apologize for responding so late.
